README.md

```diff
@@ -1 +1,70 @@
-# reactjs-projects-ziffity
\ No newline at end of file
+# Getting Started with Create React App
```

Comment:
Update Readme about this project correctky

Reply:
added readme